{
  "_annotation": {
    "cve_id": "CVE-2024-38503",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-38503.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:org.apache.syncope.client.idrepo:syncope-client-idrepo-common-ui:*:*:*:*:*:maven:*:*",
                "matchCriteriaId": "80A03D57-41C3-5745-BCC1-A54D6606E975",
                "versionEndExcluding": "3.0.8",
                "versionStartIncluding": "2.1",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      },
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:org.apache.syncope.client.idrepo:syncope-client-idrepo-console:*:*:*:*:*:maven:*:*",
                "matchCriteriaId": "4CD4BA93-ABD0-5DAF-8FD1-25B6A96955B6",
                "versionEndExcluding": "3.0.8",
                "versionStartIncluding": "2.1",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}